### DISCUSSION TITLE
We will miss you Jean-Thierry Kéchichian

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#14953] [General]
Moment of silence

This is with immense sadness that we have to announce the passing of Jean-Thierry Kéchichian [https://github.com/jtkech. Whoever has been part of our community knew and loved him. He was the most generous and talented contributor. It will never be the same without you JT.

**Comment by hishamco** [comment]
I'm shocked .. No words can express my feeling :(

**Reply by MikeAlhayek** [reply]
@hishamco yes I am still in disbelieve! Please move this discussion here https://github.com/OrchardCMS/OrchardCore/issues/14954
